packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -110,13 +110,13 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
     ): Binding<D> {
       // We have an instance of
       if (dataSource instanceof juggler.DataSource) {
-        const key = `datasources.${name || dataSource.name}`;
+        const key = `datasources.${name ?? dataSource.name}`;
```

Comment:
Before this change, an empty name `''` will be converted to `dataSource.name`. With the change in place, we will preserve the empty string and create key value `datasource.`. I think that's not correct. Thoughts?